src/main/java/com/puppycrawl/tools/checkstyle/checks/design/InnerTypeLastCheck.java

```diff
@@ -36,6 +36,26 @@
  * <pre>
  * &lt;module name=&quot;InnerTypeLast&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test {
+ *     class InnerTest1 {}
+ *     private String s; // violation, field should be declared before inner types.
+ *
+ *     class InnerTest2 {}
+ *
+ *     public void test() {} // violation, field should be declared before inner types.
+ *     interface InnerInterface {}
```

Comment:
also check will report violation even if there is only one nested type. Please remove other types and leave only one.

Reply:
Done